### DISCUSSION TITLE
Anyone using on Umbraco Cloud?

### DISCUSSION THREAD

**Discussion by leekelleher** [leekelleher/umbraco-contentment#93] [Question] [answered]
For the [project page on Our Umbraco](https://our.umbraco.com/packages/backoffice-extensions/contentment/), there is a flag for **"Works on Umbraco Cloud"**, but I've not used it myself on Cloud, so unsure if it does or doesn't. I mean, I designed Contentment for standard Umbraco use - so I'd expect it to work anywhere, but I assume that the flag is whether it works with Umbraco Deploy transfers, and of that I'm unsure. 🤷 

Has anyone used Contentment on an Umbraco Cloud project? Does it work? If it does, should I enquire to have the **"Works on Umbraco Cloud"** flag enabled? 😃

**Comment by hfloyd** [comment, marked as answer]
I'm using it successfully on Umbraco Cloud!

Specifically, I have used the Data Lists (.Net Enumeration, Umbraco Content, and Custom data sources), Render Macro, and Notes. Haven't had any issues.

**Reply by leekelleher** [reply]
Thank you @hfloyd. 🙏  Great to know. Few other people have told me privately (well Slack chats) that it works on Cloud, so I'll find out how to get the flag set on the project page. 👍

**Reply by hfloyd** [reply]
Great!

**Comment by leekelleher** [comment]
Follow up, for anyone following along at home. 👋 

I've contacted Umbraco HQ, to request that the Contentment packages can be marked at Cloud compatible.

<https://our.umbraco.com/documentation/Umbraco-Cloud/Frequently-Asked-Questions/#do-you-support-package-x-on-umbraco-cloud>

**Comment by leekelleher** [comment]
Final update. I had a nice chat with Hjalte at HQ, and now [Contentment](https://our.umbraco.com/packages/backoffice-extensions/contentment/) has been flagged as "**Works on Umbraco Cloud**"! 🎉 

_(Closing off this discussion.)_
